### ISSUE TITLE
Viewing User Record: ActionView::Template::Error: undefined method `limit' for nil:NilClass

### ISSUE THREAD

**Issue by ACPK** [thoughtbot/administrate#599]
When I try to "view" an individual user record, I get the following error:

MESSAGE:

```
ActionView::Template::Error: undefined method `limit' for nil:NilClass
```

BACKTRACE:

```
show.html.erb  37  block in _app_views_admin_vendors_show_html_erb(...)
[PROJECT_ROOT]/app/views/admin/vendors/show.html.erb:37:in `block in _app_views_admin_vendors_show_html_erb'
```

LINE CAUSING THE ISSUE:

```
# app/views/admin/vendors/show.html.erb:37 :in `block in _app_views_admin_vendors_show_html_erb`

<dd class="attribute-data attribute-data--<%=attribute.html_class%>">
  <%= render_field attribute %></dd>
<% end %>
```

**Comment by ACPK** [comment]
Has anyone else his this issue?

**Comment by ID25** [comment]
I got the same error when i have an association without any records.

**Comment by ACPK** [comment]
@ID25 @simmerer If you don't mind me asking, what's the recommended fix?

**Comment by ACPK** [comment]
@ID25 What's the best way to find the issue?

**Comment by haseeb-ahmad** [comment]
I am having the same issue here, cannot view a single record as well as cannot view collection of records..
`<%= render_field attribute %>` 
if i comment this out it runs but does not show attributes.. Does any one has any leads on how to solve this..I have been on this for a couple of days and cannot seem to find the solution..

**Comment by nickcharlton** [comment]
I've just merged #771, so this should now be fixed. I'm going to close this issue, but feel free to open another if this doesn't solve it for you.
